pkg/app/piped/cloudprovider/cloudrun/client_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudrun
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMakeCloudRunParent(t *testing.T) {
+	const projectID = "projectID"
+	got := makeCloudRunParent(projectID)
+	want := "namespaces/projectID"
+	require.Equal(t, want, got)
```

Comment:
nit: I am aware that the `require` package is used when you want to exit the function immediately, such as when you fail in the test preparation stage.

For the check you pay attention to, I feel like `assert` is suitable. Nothing changes on the result though.
```suggestion
	assert.Equal(t, want, got)
```

Reply:
Yep, It's sound better because there are no prerequisites here.
I gonna apply it :)